src/server/helpers/achievement.js

```diff
@@ -401,7 +401,19 @@ export async function getEditsInDays(orm, editorId, days) {
 		and created_at > (SELECT CURRENT_DATE - INTERVAL '${days} days');`;
```

Comment:
I think we should rename this method `getConsecutiveDaysWithEdits` and modify the JSDocs definitions to specify consecutive days and avoid possible confusion